docker-compose.yml

```diff
@@ -53,6 +53,8 @@ services:
       - RELAYER_TARGET_CHAIN_OUTPUT_FORMAT=json
       - RELAYER_TARGET_CHAIN_SIGN_MODE_STR=direct
       - RELAYER_REGISTRY_ADDRESSES=
+      - RELAYER_NEUTRON_CHAIN_ALLOW_KV_CALLBACKS=1
+      - RELAYER_ALLOW_TX_QUERIES=1
```

Comment:
why `1`, not `true`?

Reply:
dunno, bc it works 😅
shall I change it to `true`?

Reply:
Let's hope next reviewer wouldn't ask us to rename it back to 1 :v

Reply:
I believe `true` is kinda more explicit, due the type is bool

Reply:
hey guys we can save a couple of bytes by dropping the superfluous `true` and using simple `1`

Reply:
nope

Reply:
just a joke